Applications/Spire/Source/Ui/ListView.cpp

```diff
@@ -108,8 +124,9 @@ ListView::SelectionMode ListView::get_selection_mode() const {
 
 void ListView::set_selection_mode(SelectionMode selection_mode) {
   m_selection_mode = selection_mode;
-  if(m_selection_mode == SelectionMode::NONE && m_selected.has_value()) {
-    select_item(false);
+  if(m_selection_mode == SelectionMode::NONE &&
```

Comment:
This isn't in the spec, remove it.

Reply:
The purpose is when the selection mode becomes NONE from SINGLE, the selected item should clear the selected state. Otherwise, there may be a selected item even though the selection mode is NONE.

Reply:
What part of the spec says that the selected item should clear?

Reply:
Specs don't say anything about switching selection mode. I will remove it.

Reply:
In the future if you see something you think is wrong, you are always welcome to bring it up to the UX designer or to me. There could definitely be issues with the spec, bugs, or you are welcome to express your opinion and provide insight for us to learn from, but without bringing the issue up I will usually insist that things be implemented according to the spec.

In this particular case, selection mode only determines how the user interacts with the list, it doesn't determine what is selected or not selected. So for example it's possible to have a read-only selection by setting the selection mode to NONE. There could still be selected items but the user will have no way to modify what is or isn't selected.